assets/js/components/ClustersList.test.jsx

```diff
@@ -143,7 +143,8 @@ describe('ClustersList component', () => {
 
       renderWithRouter(StatefulClustersList);
 
-      expect(screen.getByText('HA1, HA2')).toBeVisible();
+      expect(screen.getByText('HA1')).toBeVisible();
+      expect(screen.getByText('HA2')).toBeVisible();
```

Comment:
Commas should be still there

Reply:
@dottorblaster see (first screenshot)->
https://github.com/trento-project/web/pull/1715/files#r1300067494

I guess that because of the svg element inbetween, the `screen.getByText` is not catching it. Is it ok to leave the test like this?

Reply:
Thanks @rtorrero, it's ok.